### PR TITLE
Update virus-initiative-criteria.md

### PR DESCRIPTION
**Pull request by adamoboe**
Removed the notice that the program was not accepting applications

### CODE REVIEW

**Comment by adamoboe** [comment]
Please approve the request. MVI is now open for new applications as of June 1.

File: microsoft-365/security/intelligence/virus-initiative-criteria.md [hunk 1]
```diff
@@ -26,8 +26,6 @@ The Microsoft Virus Initiative (MVI) helps organizations develop better-together
 
 ## Become a member
 
-> [!NOTE]
-> The MVI Program is not currently accepting new applications for membership. Please contact MVI@microsoft.com for more information.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/intelligence/virus-initiative-criteria.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @adamoboe, kindly commit the suggested change to remove the extra blank line. Many thanks.

**Review by denisebmsft — APPROVED** [review]
Thank you @adamoboe @JohanFreelancer9

**Pull request merged**